.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,262 @@
+name: Publish release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  publish-release:
+    name: Publish Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: '11'
```

Comment:
@NipunaMadhushan Since 1.2.x support JDK8 lets we use that to build it?

Reply:
@keizer619 sure.